pegasus/sites.v3/code.org/public/starwars.haml

```diff
@@ -18,6 +18,7 @@ social:
   "og:video:width": ""
   "og:video:height": ""
   "og:video:type": ""
+video_player: true
```

Comment:
/starwars video player already works.  Any idea why this line helps on some of these pages?